### PR TITLE
Search Applicants by Notes

### PR DESCRIPTION
**Pull request by YuvBindal**

### CODE REVIEW

**Review by YuvBindal — COMMENTED** [review]
Need to fix checkstyle issues

**Review by YuvBindal — COMMENTED** [review]
Should be good to merge other wise?

**Review by YuvBindal — COMMENTED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/SearchApplicantCommand.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        Predicate<Person> matchesCriteria = person -> {
+            if (!(person instanceof Applicant)) {
+                return false;
+            }
+            Applicant applicant = (Applicant) person;
+
+            // Check if the roleName matches filteredRole and stageName matches filteredStage
```
Comment (ys112) [inline src/main/java/seedu/address/logic/commands/SearchApplicantCommand.java:49]: Comment does not seem to match context here too.

**Review by ys112 — CHANGES_REQUESTED** [review]
Would like to bring your attention to a couple of things to take note of:
1. I tried running the app and it would not show 2 applicants with the same keyword.
2. Highly suggest fixing the check style issues first before merging as the CI is crucial to ensure our app can be built and catch any issues in the earlier stage.
3. Seems like your code is not up to date from the latest master commit from the team's tp as I noticed the notes feature is still before the fix.

Steps to replicate issue:
1. Have 2 applicants with "forbes" in notes
4. search /notes forbes
5. List empty

**Pull request closed without merging**